app/src/main/java/org/kiwix/kiwixmobile/webserver/ZimHostFragment.kt

```diff
@@ -82,6 +82,7 @@ class ZimHostFragment : BaseFragment(), ZimHostCallbacks, ZimHostContract.View {
   private var ip: String? = null
   private lateinit var serviceConnection: ServiceConnection
   private var progressDialog: ProgressDialog? = null
+  private lateinit var booksList: List<BooksOnDiskListItem>
```

Comment:
@macgills It crashes if we don't select the books. How do I solve this?

Reply:
Wrapping it in a check like `selectedBooksPath.size > 0` should do the trick. I believe